framework/src/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -505,6 +518,29 @@ private void parseMethod(MethodDeclaration decl, ExecutableElement elt,
         putNew(atypes, elt, methodType);
     }
 
+    private void handleExistingAnnotations(AnnotatedTypeMirror atype, Type typeDef) {
```

Comment:
I meant that this method needs Javadoc.

However, you are right that the behavior should be documented in the Checker Framework manual as well.  I suggest a paragraph at the end of section "Using a stub file".